src/StatusQ/Core/Utils/Utils.qml

```diff
@@ -75,6 +75,73 @@ QtObject {
     function isHtml(text) {
         return (/<\/?[a-z][\s\S]*>/i.test(text))
     }
+
+    // function to draw arrow
+    function drawArrow(context, fromx, fromy, tox, toy, color) {
+        const dx = tox - fromx;
+        const dy = toy - fromy;
+        const headlen = 10; // length of head in pixels
+        const angle = 0
+        const radius = 5
+
+        context.strokeStyle = color ? color : '#627EEA'
```

Comment:
Shouldn't that color come from a Theme? Will it always be the same for light/dark?